docs/src/concepts.md

```diff
@@ -24,21 +32,37 @@ Over the several years, many labs have developed DataJoint-based pipelines for l
-| ------------------ | ---------------------------------------------------------------- |
-| Lab                | Table for storing general lab info.                              |
-| Location           | Location of research (e.g., animal housing or experimental rigs) |
-| UserRole           | Roles assigned to a user or a job title.                         |
-| User               | Table for storing user information.                              |
-| LabMembership      | Store lab membership information using three lookup tables.      |
-| ProtocolType       | Type of protocol or issuing agency.                              |
-| Protocol           | Protocol specifics (e.g., protocol number and title).            |
-| Project            | Projects within a lab.                                           |
-| ProjectKeywords    | Project keywords or meta-information.                            |
-| ProjectPublication | Project's resulting publications.                                |
-| ProjectSourceCode  | URL to source code for replication.                              |
-| ProjectUser        | Users participating in the project.                              |
-| Source             | Source or supplier of subject animals.                           |
\ No newline at end of file
+| Table                 | Description                                                  |
+| ------------------    | -------------------------------------------------------------|
+| Lab                   | Table for storing general lab info.                          |
+| Location              | Location of research (e.g., animal housing, experimental rigs)|
+| UserRole              | Roles assigned to a user or a job title.                     |
+| User                  | Table for storing user information.                          |
+| LabMembership         | Store lab membership information using three lookup tables.  |
+| ProtocolType          | Type of protocol or issuing agency.                          |
+| Protocol              | Protocol specifics (e.g., protocol number and title).        |
+| Project[^1]           | Projects within a lab.                                       |
+| ProjectKeywords[^1]   | Project keywords or meta-information.                        |
+| ProjectPublication[^1]| Project's resulting publications.                            |
+| ProjectSourceCode[^1] | URL to source code for replication.                          |
+| ProjectUser[^1]       | Users participating in the project.                          |
+| Source                | Source or supplier of subject animals.                       |
+
+[^1]: 
+    Depreciation warning. A subset of the tables above will be removed in a future
+    version of this Element in favor of the `project` schema.
+
+### `project` schema ([API docs](https://datajoint.com/docs/elements/element-lab/api/element_lab))
```

Comment:
```suggestion
### `project` schema ([API docs](../api/element_lab/project))
```

Reply:
Please verify that this is the correct implementation

Reply:
Thanks @kabilar - I've confirmed that similar routing works in a local deployment and added this in my most recent commit